.travis.yml

```diff
@@ -1,29 +1,33 @@
 language: python
 sudo: false
-python: 2.7
 env:
   global:
     - PLANEMO_SKIP_REDUNDANT_TESTS=1
     - PLANEMO_ENABLE_POSTGRES_TESTS=1
     - PLANEMO_SKIP_GALAXY_CWL_TESTS=1
     - PLANEMO_TEST_WORKFLOW_RUN_PROFILE=travisworkflowtests
-  matrix:
-    - TOX_ENV=py27-lint
-    - TOX_ENV=py34-lint
-    - TOX_ENV=py27-lint-readme
-    - TOX_ENV=py27-lint-docs
-    - TOX_ENV=py34-quick
-    - TOX_ENV=py27
-    - TOX_ENV=py34
-    - TOX_ENV=py27-lint-docstrings
-
-install:
-  - pip install tox coveralls
-
 matrix:
+  include:
+    - python: 2.7
+      env: TOX_ENV=py27-lint
+    - python: 2.7
+      env: TOX_ENV=py27-lint-readme
+    - python: 2.7
+      env: TOX_ENV=py27-lint-docs
+    - python: 2.7
+      env: TOX_ENV=py27-lint-docstrings
+    - python: 2.7
+      env: TOX_ENV=py27
+    - python: 3.6
+      env: TOX_ENV=py36-quick
```

Comment:
It would be useful to keep `py3?-lint` I think, since flake8 performs slightly different checks for the various Python versions.

Also, maybe put `py36-quick` before `py27` again, to have a faster feedback.